rising/transforms/spatial.py

```diff
@@ -82,33 +83,13 @@ def forward(self, **data) -> dict:
             dict with augmented data
         """
         if torch.rand(1) < self.prob:
-            k = random.randrange(0, 4)
-            rand_dims = self._permutations[random.randrange(0, len(self._permutations))]
+            num_rots = self.num_rots
+            rand_dims = self.dims
 
             for key in self.keys:
-                data[key] = rot90(data[key], k=k, dims=rand_dims)
+                data[key] = rot90(data[key], k=num_rots, dims=rand_dims)
         return data
 
-    @property
-    def dims(self) -> Sequence:
-        """
-        Number of dimensions
-        """
-        return self._dims
-
-    @dims.setter
-    def dims(self, dims: Sequence):
-        """
-        Set number of dimensions and compute new set of permutations
-
-        Parameters
-        ----------
-        dims: Sequence
-            number of dimensions
-        """
-        self._dims = dims
-        self._permutations = tuple(permutations(dims, 2))
-
 
 class Resize(BaseTransform):
     def __init__(self, size: Union[int, Sequence[int]], mode: str = 'nearest',
```

Comment:
We should apply the same solution as in our affine rotation to the `size` parameter.